LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 onetask.ai GmbH
+   Copyright 2024 Kern AI GmbH
```

Comment:
year shouldn't be changed as it indicates the [start period](https://stackoverflow.com/a/2391555) if i understood it right. Not sure how this applies to name changes as Kern AI GmbH didn't exist in 2022

Reply:
i would think it's `2022 Kern AI`

Reply:
can be ignored